tools/topology/topology2/include/components/src_format_s32_to_sxx_convert.conf

```diff
@@ -1,346 +1,57 @@
-				{
-					out_rate                 64000
-					out_bit_depth            32
-					out_valid_bit_depth      24
-				}
-				# 88.2khz output
-				{
-					out_rate                 88200
-					out_bit_depth            32
-					out_valid_bit_depth      24
-				}
-				# 96khz output
-				{
-					out_rate                 96000
-					out_bit_depth            32
-					out_valid_bit_depth      24
-				}
-				# 176.4khz output
-				{
-					out_rate                 176400
-					out_bit_depth            32
-					out_valid_bit_depth      24
-				}
-				# 192khz output
-				{
-					out_rate                 192000
-					out_bit_depth            32
-					out_valid_bit_depth      24
-				}
-				# 8khz output
-				{
-					out_rate		 8000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 11.025 khz output
-				{
-					out_rate                 11025
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 12khz output
-				{
-					out_rate                 12000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 16khz output
-				{
-					out_rate                 16000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 22.05khz output
-				{
-					out_rate                 22050
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 24khz output
-				{
-					out_rate                 24000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 32khz output
-				{
-					out_rate                 32000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 44.1khz output
-				{
-					out_rate                 44100
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 48khz output
-				{
-					out_rate                 48000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 64khz output
-				{
-					out_rate                 64000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 88.2khz output
-				{
-					out_rate                 88200
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 96khz output
-				{
-					out_rate                 96000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 176.4khz output
-				{
-					out_rate                 176400
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 192khz output
-				{
-					out_rate                 192000
-					out_bit_depth            32
-					out_valid_bit_depth      32
+			CombineArrays.Object.Base.output_audio_format [
+				{
+					out_rate [
+						8000
+						11025
+						12000
+						16000
+						22050
+						24000
+						32000
+						44100
+						48000
+						64000
+						88200
+						96000
+						176400
+						192000
+					]
+					out_bit_depth [
+						16
+						32
+					]
+					out_valid_bit_depth [
+						16
+						24
+						32
```

Comment:
@btian1 we cant do this. This would make several invalid combinations such as 16 for bit_depth and 23 for valid_bit_depth. So please separate them out into 3 different combinearray definitions

Reply:
if separate, is it 3 or 6? since out_bit_depth also need a separation?
Also, will the invalid combination will be used with error or just put it there no use?

Reply:
you might have to figure out the right combination so that we dont end up with invalid audio formats and the exact number as there is currently